### PR TITLE
libsndfile: Update to 1.0.29pre2

### LINKED ISSUE #6618: libsndfile: needs update to 1.0.29pre2 for pkg-config fixes
Upstream issue: https://github.com/erikd/libsndfile/pull/515

Upstream is trying to get a release out for a year now so waiting might be futile.

CC @andlabs

### PR DESCRIPTION
**Pull request by dofuuz**
Resolves  #6618

### CODE REVIEW

**Comment by dofuuz** [comment]
Note:  `--enable-stack-smash-protection` is workaround for issue #5803.
erikd/libsndfile#532 didn't worked for me.

File: mingw-w64-libsndfile/PKGBUILD [hunk 1]
```diff
@@ -3,27 +3,23 @@
...
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=1.0.28
+pkgver=1.0.29pre2
 pkgrel=1
 pkgdesc="A C library for reading and writing files containing sampled sound (mingw-w64)"
 arch=('any')
 url="http://www.mega-nerd.com/libsndfile"
 license=('LGPL')
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc")
-depends=("${MINGW_PACKAGE_PREFIX}-flac" "${MINGW_PACKAGE_PREFIX}-libvorbis" "${MINGW_PACKAGE_PREFIX}-speex")
```
Comment (lazka) [inline mingw-w64-libsndfile/PKGBUILD:13]: why the speex removal?

Comment (dofuuz) [reply mingw-w64-libsndfile/PKGBUILD:13]: To use speex, --enable-experimental should be added.

**Comment by Biswa96** [comment]
Just a question. Is it possible to use CMake to build the project? CMake options are simple. Also the CMake output shows some other dependnecies which are not present here.

**Comment by dofuuz** [comment]
@Biswa96 Yes, but they say it's experimental.

**Comment by Biswa96** [comment]
OK.

File: mingw-w64-libsndfile/PKGBUILD (hunk 1 above)
Comment (lazka) [reply mingw-w64-libsndfile/PKGBUILD:13]: ah, I see, thanks. speex isn't that important anymore anyway

**Pull request merged**